code/modules/mob/living/carbon/human/species/golem.dm

```diff
@@ -7,6 +7,7 @@
 	default_language = "Galactic Common"
 	flags = NO_BREATHE | NO_PAIN | NO_BLOOD | NO_SCAN
 	dietflags = DIET_OMNI		//golems can eat anything because they are magic or something
+	reagent_tag = PROCESS_DUO	//golems only process a select few chems like acid and hellwater because magic?
```

Comment:
This is a noticeable nerf to golems. Unless I am missing something, it means that if they take any brute/burn damage, it cannot be healed, and if they ever fall into crit they are doomed.